src/views/BuildingCreateView.vue

```diff
@@ -0,0 +1,100 @@
+<script setup lang="ts">
+import { ref } from 'vue';
+import { useRouter, useRoute } from 'vue-router';
+import axios from 'axios';
+import ReusableFormComponent from '@/components/ReusableFormComponent.vue';
+
+// Router und Route für IDs
+const router = useRouter();
+const route = useRoute();
+const projectId = route.params.projectId as string;
+const propertyId = route.params.propertyId as string;
+
+// Typ für Felder
+interface Field {
+  name: string;
+  label: string;
+  type: "text" | "textarea" | "checkbox" | "select";
+  options?: any[];
+  required?: boolean;
+  validations?: ((value: any) => string | null)[];
+}
+
+// Form Felder
+const fields: Field[] = [
+  { name: 'title', type: 'text', label: 'Building Name', required: true },
+  { name: 'addressId', type: 'text', label: 'Address ID', required: true },
+  { name: 'description', type: 'textarea', label: 'Description' },
+  {
+    name: 'commercialSpace',
+    type: 'text',
+    label: 'Commercial Space (m²)',
+    required: true,
+    validations: [(value) => isNaN(Number(value)) ? 'Must be a number' : null],
+  },
+  {
+    name: 'usableSpace',
+    type: 'text',
+    label: 'Usable Space (m²)',
+    required: true,
+    validations: [(value) => isNaN(Number(value)) ? 'Must be a number' : null],
+  },
+  {
+    name: 'heatingSpace',
+    type: 'text',
+    label: 'Heating Space (m²)',
+    validations: [(value) => isNaN(Number(value)) ? 'Must be a number' : null],
+  },
+  {
+    name: 'rent',
+    type: 'text',
+    label: 'Rent (€)',
+    validations: [(value) => isNaN(Number(value)) ? 'Must be a number' : null],
+  },
+];
+
+// Ladezustand
+const isLoading = ref(false);
+
+// Daten senden
+const handleSubmit = async (formData: Record<string, any>) => {
+  isLoading.value = true;
+  try {
+    await axios.post(`/project/${projectId}/property/${propertyId}/buildings`, formData);
```

Comment:
Correct routing format, but there is further nesting (hierarchy), which makes it different from the /project/:projectId/commercial/:commercialId template. Is there a reason for this further nesting?